### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by ejkitchen** [openai/whisper#171] [General]
When running on audio files with voices, your model is impressive, great work. But using the medium English model on audio with no voices and low ambient background noise, I sometimes get "Thank you for watching!". I listened to the audio, and there is absolutely nothing in there that could even be construed as that. The audio file was only 2 min long. Just an FYI

**Comment by jongwook** [comment]
This is an example of hallucination mentioned in [the model card](https://github.com/openai/whisper/blob/main/model-card.md). This can be mitigated by tuning the VAD hyperparameters (`--compression_ratio_threshold`, `--logprob_threshold`, `--no_speech_threshold`), and we are looking for ways to further prevent hallucination. Meanwhile, combining with an external VAD model seems to be a good option as well #29.
